mix.exs

```diff
@@ -36,7 +36,7 @@ defmodule Companies.MixProject do
   defp deps do
     [
       {:appsignal, "~> 1.0"},
-      {:bamboo, "~>1.2.0"},
+      {:bamboo, github: "thoughtbot/bamboo"},
```

Comment:
aaaah so the one on master supports dynamic templates!

Reply:
This has now been released as `1.3`